woeip/apps/core/models.py

```diff
@@ -1,6 +1,123 @@
 from django.contrib.auth.models import AbstractUser
+from django.contrib.gis.db.models import LineStringField, PointField
+from django.contrib.postgres.fields import JSONField
+from django.db import models
 
 
 class User(AbstractUser):
     class Meta:
         get_latest_by = 'date_joined'
+
```

Comment:
Consider moving these into a separate app to avoid having too many models in one app. This keeps the code legible.

Reply:
Are you suggesting give the ``User`` model in its own app ``core`` and move everything else to a new app?

Reply:
I’m suggesting moving the new models to their own app.